src/validations/commentValidation.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
+import Joi from 'joi';
+
+const validateComment = Joi.object().keys({
+  comment: Joi.string().required().label('a Comment required')
```

Comment:
@SergeNm You can also give it a minimum of words to comment for example two words or a full word.